flowtools/upload_files/upload.xml

```diff
@@ -0,0 +1,235 @@
+  
```

Comment:
I'm confused... isn't this the normal Galaxy upload tool? This should be already in the instance... at the very top.

Reply:
in our galaxy instance, at the very top in the tool panel, if you press the icon with the arrow pointing up, it should be the same tool.

Reply:
Hi Pablo. Immport has a different upload tool which can download fcs files. The tool currently in our instance doesn't have an option to detect files ending with fcs (and other flow related objects) and when I try to upload fcs files it fails. Long term I think it would be useful to combine the two tools, but as I don't know python (which the upload tools are written in) I thought it would be easier for now to just add the immport upload tool as well. Let me know if I'm missing something obvious and there's a better to way to go about this! @pcm32 
![Annotation 2020-05-19 103844](https://user-images.githubusercontent.com/52698020/82310950-f4e87d80-99bc-11ea-9e20-eb02f319ceb0.png)

Reply:
Ok, I see. Probably the correct way to go on this is to add the Galaxy FCS datatype (and any other datatype that they might be relying on) to both the main Galaxy repo and to our setups while it is not in a Galaxy release. Then that will show up in the normal Galaxy upload and we don't need an extra upload. This will also mean that the datatype will be used in other parts. Thanks!

Reply:
@pcm32 ok, is that something I will be able to do? If so, would you have some time to show me how to add those other datatypes. Thank you.